Lib/asyncio/base_events.py

```diff
@@ -277,7 +277,8 @@ def __init__(self, loop, sockets, protocol_factory, ssl_context, backlog,
                  ssl_handshake_timeout, ssl_shutdown_timeout=None):
         self._loop = loop
         self._sockets = sockets
-        self._active_count = 0
+        # Weak references so abandoned transports can be detected
```

Comment:
```suggestion
        # Weak references so abandoned transports can be ignored
```

Reply:
The wording here was intentional. Weak references is to my knowledge the only way to detect abandoned objects. But it's not this code that does that detection, so I can understand the confusion. How about:

> Weak references so we don't break Transport's ability to detect abandoned transports

?